peps/pep-0639.rst

```diff
@@ -117,317 +86,395 @@ Motivation
-  a project used for development, while an **installed project** is the form a
-  project takes once installed from a distribution, as
-  `specified by PyPA <installedspec_>`__.
-
-**Project Source Metadata** *(Sub: Project Table Metadata, Key, Subkey)*
-  Core metadata defined by the package author in the project source tree,
-  as top-level keys in the ``[project]`` table of a ``pyproject.toml`` file,
-  in the ``[metadata]`` table of ``setup.cfg``, or the equivalent for other
-  build tools.
-
-  The **Project Table Metadata**, or ``pyproject.toml`` ``[project]`` metadata,
-  refers specifically to the former, as defined by the
-  `PyPA Declaring Project Metadata specification <pep621spec_>`__
-  and originally specified in :pep:`621`.
-  A **Project Table Key**, or an unqualified *key* refers specifically to
-  a top-level ``[project]`` key
-  (notably, distinct from a core metadata *field*),
-  while a **subkey** refers to a second-level key in a table-valued
-  ``[project]`` key.
-
-**Root License Directory** *(Short: License Directory)*
-  The directory under which license files are stored in a project/distribution
-  and the root directory that their paths, as recorded under the
-  ``License-File`` core metadata fields, are relative to.
-  Defined here to be the project root directory for source trees and source
-  distributions, and a subdirectory named ``licenses`` of the directory
-  containing the core metadata (i.e., the ``.dist-info/licenses``
-  directory) for built distributions and installed projects.
-
-**Tool** *(Sub: Packaging Tool, Build Tool, Install Tool, Publishing Tool)*
-  A program, script or service executed by the user or automatically that
-  seeks to conform to the specification defined in this PEP.
-
-  A **packaging tool** refers to a tool used to build, publish,
-  install, or otherwise directly interact with Python packages.
-
-  A **build tool** is a packaging tool used to generate a source or built
-  distribution from a project source tree or sdist, when directly invoked
-  as such (as opposed to by end-user-facing install tools).
-  Examples: Wheel project, :pep:`517` backends via ``build`` or other
-  package-developer-facing frontends, calling ``setup.py`` directly.
-
-  An **install tool** is a packaging tool used to install a source or built
-  distribution in a target environment. Examples include the PyPA pip and
-  ``installer`` projects.
-
-  A **publishing tool** is a packaging tool used to upload distribution
-  archives to a package index, such as Twine for PyPI.
-
-**Wheel** *(Short: wheel, Rel: wheel format, Wheel project)*
-  Here, **wheel**, the standard built distribution format introduced in
-  :pep:`427` and `specified by the PyPA <wheelspec_>`__, will be referred to in
-  lowercase, while the `Wheel project <wheelproject_>`__, its reference
-  implementation, will be referred to as such with **Wheel** in Title Case.
+
+.. _639-terminology-packaging:
+
+Packaging terms
```

Comment:
This section feels like it should be a standalone PR to the packaging spec directly.

Reply:
I had similar feeling. Could we do that, @CAM-Gerlach?

Reply:
Done in https://github.com/pypa/packaging.python.org/pull/1541; references changed in c3c6e446381667b68d777404a093f75d49f3e3d2